twentytwentythree/functions.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
It is a bummer that we need to add this whole file just to get font-smoothing. Do you foresee any other CSS that will need to be added? 

Otherwise maybe we can instead push for a theme.json setting that opts-in to font-smoothing.